.github/workflows/main.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         node-version: [18]
-        pnpm-version: [7]
+        pnpm-version: [7.19]
```

Comment:
Latest version of `pnpm` seems [broken](https://github.com/pnpm/pnpm/issues/5845).